### PR TITLE
Clarify Android wording and ref latest versions

### PR DESCRIPTION
**Pull request by jamesonwilliams**
1. Shorten some run on sentences, and otherwise simplify expressions for clarity.
2. Recommend the use of the latest versions of Gradle Plugin, Amplify Modelgen plugin, etc.
3. Update some factual info, for example, minSdk has become 16.
4. Fix indentation in Gradle snippet(s)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/lib/getting-started/fragments/android/setup.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-Build an Android app using the Amplify Framework which contains:
+Build an Android app using the Amplify Framework. The Amplify Framework includes:
```
[the remark this answers is not in the document]
Comment (jpignata) [reply docs/lib/getting-started/fragments/android/setup.md]: After some thought, I do think it's a worthwhile goal, but we need to DRY up the content a bit to enable it. On @brene's post-launch list.

File: docs/lib/getting-started/fragments/android/setup.md [hunk 2]
```diff
@@ -42,37 +42,37 @@ buildscript {
...
+        classpath 'com.android.tools.build:gradle:3.6.1'
         classpath 'com.amplifyframework:amplify-tools-gradle-plugin:0.2.1'
     }
 }
 
 apply plugin: 'com.amplifyframework.amplifytools'
 ```
 
-b. Next, add the following dependencies to your **app** `build.gradle` and `compileOptions` to work with the Java 8 features used:
+b. Next, add the following dependencies to your **app** `build.gradle`. Note the declaration of `compileOptions`, to make use of Java 8 features:
```
[the remark this answers is not in the document]
Comment (jamesonwilliams) [reply docs/lib/getting-started/fragments/android/setup.md]: Yes! Agree. Will add.

File: docs/lib/getting-started/fragments/android/setup.md [hunk 3]
```diff
@@ -16,24 +16,24 @@ This page guides you through setting up a backend and integration into your Andr
...
-* If you had previously installed Amplify CLI (< 4.5.0), update to the latest version by running:
+* If you have previously installed Amplify CLI (< 4.5.0), update to the latest version by running:
 
 ```terminal
 $ npm install -g @aws-amplify/cli
 ```
 
 ## Step 1: Configure your app
-You can use an existing Android app or create a new Android app in Java as per the steps in prerequisite section.
+You can use an existing Android app or create a new Android app in Java as per the steps in the [prerequisites section](#prerequisites).
```
[the remark this answers is not in the document]
Comment (jamesonwilliams) [reply docs/lib/getting-started/fragments/android/setup.md]: Fair points. Will rework this.

File: docs/lib/getting-started/fragments/android/setup.md [hunk 4]
```diff
@@ -16,24 +16,24 @@ This page guides you through setting up a backend and integration into your Andr
...
-* [Install Android Studio](https://developer.android.com/studio/index.html#downloads) version 3.1 or higher.
+* [Install Android Studio](https://developer.android.com/studio/index.html#downloads) version 3.6 or higher.
 
-* [Install Android SDK with a minimum API level of 15 (Ice Cream Sandwich).](https://developer.android.com/studio/releases/platforms)
+* [Install Android SDK with a minimum API level of 16 (Jelly Bean).](https://developer.android.com/studio/releases/platforms)
 
 * This guide assumes that you are familiar with Android development and tools. If you are new to Android development, you can follow [these steps](https://developer.android.com/training/basics/firstapp/creating-project) to create your first Android application using Java.
 
-* If you had previously installed Amplify CLI (< 4.5.0), update to the latest version by running:
+* If you have previously installed Amplify CLI (< 4.5.0), update to the latest version by running:
```
[the remark this answers is not in the document]
Comment (jamesonwilliams) [reply docs/lib/getting-started/fragments/android/setup.md]: I agree. I think this is language is a remnant of the churn around the time of re:Invent launch. Will reword it to introduce the CLI as a new actor.

File: docs/lib/getting-started/fragments/android/setup.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jamesonwilliams) [reply docs/lib/getting-started/fragments/android/setup.md]: I'm not sure how best we can share content across the platforms. A lot of the shared language is intertwined in platform-specific wording and code samples. I'd love to say "we'll always update all the platforms," or "we could use shared snippets." Or something like this. But, I can't convince myself that the overhead is worth it, in either case. The KISS approach of disjoint docs might be most frugal.
